### DISCUSSION TITLE
Equivalent to fp-ts flow?

### DISCUSSION THREAD

**Discussion by nkiesel** [arrow-kt/arrow#3009] [Q&A] [answered]
fp-ts has [flow](https://gcanti.github.io/fp-ts/modules/function.ts.html#flow) which allows to decompose nested calls.  is there something similar in Arrow?

**Comment by kyay10** [comment, marked as answer]
There's no real need for this in Kotlin. Lambda syntax suffices:
```kotlin
{ a: A -> a.let(ab).let(bc).let(cd) ... }
```
